.github/actions/terraform-observe/opal-validation/main.tf.original

```diff
@@ -0,0 +1,20 @@
+data "observe_workspace" "default" {
+  name = "Default"
+}
+
```

Comment:
I think we're most likely going to run into OPAL breakages outside of the module you're developing, we should discuss how we can terraform a larger project to test the inputs and outputs

Reply:
hmm, is this a concern specific to the `observe_workspace` and that we should use more than just the `Default` one? If so, would it be possible / hard to just input "all the observe_workspaces!"?)